jre_emul/android/platform/libcore/ojluni/src/main/java/java/util/concurrent/atomic/AtomicReferenceFieldUpdater.java

```diff
@@ -101,11 +102,8 @@
     public static <U,W> AtomicReferenceFieldUpdater<U,W> newUpdater(Class<U> tclass,
                                                                     Class<W> vclass,
                                                                     String fieldName) {
-        return new AtomicReferenceFieldUpdaterImpl<U,W>(tclass, vclass, fieldName, null);
-        /* J2ObjC: Call stack not available.
         return new AtomicReferenceFieldUpdaterImpl<U,W>
-            (tclass, vclass, fieldName, Reflection.getCallerClass()); // android-changed
-        */
+            (tclass, vclass, fieldName, Reflection.getCallerClass());
```

Comment:
Ditto: restore libcore comment.

Reply:
The "// android-changed" comment is only in libcore-nougat but not libcore